packages/overlay/overlay-trigger.md

```diff
@@ -117,6 +117,40 @@ The delivery of hover content can be customized via the `placement` attribute. H
 </overlay-trigger>
 ```
 
+### Performance optimization
```

Comment:
I don't know whether this should be called Perf Optimization but we can specify or educate the users why we should use this?